### PR TITLE
Generate ptp4l/phc2sys process status metrics 

### PR DESCRIPTION
**Pull request by aneeshkp**
Generate ptp4l/phc2sys process status  metrics  and report  status to sidecar when event  framework is  enabled

Adding health check metrics for the PTP4l process and Phc2sys process.
Generate metrics for process status and send ptp4l status and pch2sys status to event framework.

Signed-off-by: Aneesh Puttur <aneeshputtur@gmail.com>

### CODE REVIEW

File: pkg/daemon/metrics.go [hunk 1]
```diff
@@ -367,6 +385,17 @@ func UpdateInterfaceRoleMetrics(process string, iface string, role ptpPortRole)
 		"process": process, "node": NodeName, "iface": iface}).Set(float64(role))
 }
 
+func UpdateProcessStatusMetrics(process string, cfgName string, status int64) {
```
Comment (josephdrichard) [inline pkg/daemon/metrics.go:388]: Can you define "up" and "down" constants that you pass to UpdateProcessStatusMetrics as the status instead of an int64?

Comment (aneeshkp) [reply pkg/daemon/metrics.go]: Done

**Review by josephdrichard — APPROVED** [review]
/lgtm
/approved

**Pull request merged**